labrad/node/__init__.py

```diff
@@ -511,7 +501,7 @@ def __init__(self, nodename, host, port, credential):
         self.credential = credential
         self.servers = {}
         self.instances = {}
-        self.initMessages(True)
+        self.instances_to_notify = weakref.WeakSet()
```

Comment:
Class docs here would help too, in particular what are `instances` and `instances_to_notify`?